classes/network/Connection.py

```diff
@@ -89,8 +93,19 @@ def packet_chat_message(self, buff):
         p_text = buff.unpack_string()
         message = "<{0}> {1}".format(self.display_name, p_text)
         logging.info(message)
-        NetworkController.send_packet(packet_type=PacketType.CHAT_MESSAGE,
-                                      message=message)
+        out = event.fire("chat",self.display_name,p_text)
+        delete_invocation = False
```

Comment:
interesting variable name, but I believe that `suppress_packet` is better suited